src/client/src/system/service/autobahnConnectionProxy.js

```diff
@@ -9,11 +9,23 @@ export default class AutobahnConnectionProxy {
   connection:autobahn.Connection;
 
   constructor(url:string, realm:string) {
+    const useSecure = location.protocol === 'https:';
+    const securePort = 443;
+    const defaultPort = 8080;
     this.connection = new autobahn.Connection({
-      url: url,
       realm: realm,
       use_es6_promises: true,
-      max_retries: -1 // unlimited retries
+      max_retries: -1, // unlimited retries,
+      transports: [
```

Comment:
According to the Autobahn documentation, 

> supported transports are WebSocket and HTTP long-poll

cf. http://autobahn.ws/js/reference.html#connection-options